docs/general/builtins/scalar-functions.rst

```diff
@@ -3239,6 +3239,40 @@ Begin the search from given position (optional).
     ``array_position`` won't even when used inside the ``WHERE`` clause.
 
 
+.. _scalar-array_prepend:
+
+``array_prepend(value, anyarray)``
+----------------------------------
+
+The ``array_prepend`` function prepends a value to the beginning of the array.
+
+Returns: ``array``
+
+::
+
+    cr> select
+    ...     array_prepend(1, [2,3,4]) AS array_prepend;
+    +---------------+
+    | array_prepend |
+    +---------------+
+    | [1, 2, 3, 4]  |
+    +---------------+
+    SELECT 1 row in set (... sec)
+
+
+You can also use the concat :ref:`operator <gloss-operator>` ``||`` to prepend
+values to an array::
+
+    cr> select
+    ...    1 || [2,3,4] AS array_prepend;
+    +---------------+
+    | array_prepend |
+    +---------------+
+    | [1, 2, 3, 4]  |
+    +---------------+
+    SELECT 1 row in set (... sec)
+
+
```

Comment:
could we add a NOTE about the difference in null handling between function call and operator?

Reply:
done, see https://github.com/crate/crate/pull/16333/commits/aaff072332ac8698cba9ac5b4e264b175315f367